src/Sylius/Bundle/WebBundle/Controller/Frontend/Review/ProductReviewController.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\WebBundle\Controller\Frontend\Review;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use FOS\RestBundle\Controller\FOSRestController;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * @author Justin Hilles <justin@1011i.com>
+ * @author Daniel Richter <nexyz9@gmail.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ProductReviewController extends FOSRestController
+{
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function indexAction(Request $request)
+    {
+        $product = $this->findProductOr404($request->attributes->get('slug'));
+        $reviews = $this->getProductReviewRepository()->findBy([
+            'reviewSubject' => $product,
+            'status' => ReviewInterface::STATUS_ACCEPTED,
+        ]);
+
+        $view = $this
+            ->view()
+            ->setTemplate('SyliusWebBundle:Frontend/Review:_list.html.twig')
+            ->setData([
+                'product_reviews' => $reviews,
+            ])
+        ;
+
+        return $this->handleView($view);
+    }
+
+    /**
+     * @return Response
+     */
+    public function createAction()
```

Comment:
```
sylius_product_review_create:
    path: /products/{productId}/reviews/
    methods: [GET, POST]
    defaults:
        _controller: sylius.controller.product_review:createAction
        _sylius:
            factory:
                method: createForSubject
                arguments: [$productId]
```

Will replace both of these actions and works with Ajax calls too. This controller can be removed.